frontend/src/pages/Home.tsx

```diff
@@ -80,10 +85,37 @@ function Home() {
     };
   }, []);
 
+  let refreshApprovals = true;
+
+  useEffect(() => {
+    const fetchApprovals = async () => {
+      try {
+        if (refreshApprovals) {
+          setApprovals(await new BackendClient().getNewFundsPullPreApprovals());
+        }
+        setTimeout(fetchApprovals, REFRESH_APPROVALS_INTERVAL);
+      } catch (e) {
+        console.error(e);
+      }
+    };
+
+    // noinspection JSIgnoredPromiseFromCall
+    fetchApprovals();
+
+    return () => {
+      refreshApprovals = false;
+    };
+  }, []);
+
   if (!user) {
     return <WalletLoader />;
   }
 
+  const bottomStyles = {
```

Comment:
This styles could be inlined without `classNames`

Reply:
@vladkucherov - I tried to add "style={{"list-group-item": true, "text-center": true,}}" instead of "className={classNames(bottomStyles)}", but he claim "TS2322: Type '{ "list-group-item": boolean; "text-center": boolean; }' is not assignable to type 'CSSProperties'.   Object literal may only specify known properties, and '"list-group-item"' does not exist in type 'CSSProperties'."